tests/test_raster.py

```diff
@@ -3072,8 +3092,19 @@ def test_save(self, mask: gu.Mask) -> None:
         # Save file to temporary file, with defaults opts
         temp_file = os.path.join(temp_dir.name, "test.tif")
         mask.save(temp_file)
-        saved = gu.Raster(temp_file)
-        assert mask.astype("uint8").raster_equal(saved)
+        saved = gu.Mask(temp_file)
+
+        # TODO: Generalize raster_equal for masks?
```

Comment:
does it not work automatically for Masks?

Reply:
Opened #466 and looking into it...